auto_nag/scripts/regression_without_regressed_by.py

```diff
@@ -0,0 +1,207 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import dateutil.parser
+from libmozdata.bugzilla import Bugzilla, BugzillaUser
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag import utils
+
+
+class RegressionWithoutRegressedBy(BzCleaner):
+    def __init__(self):
+        super(RegressionWithoutRegressedBy, self).__init__()
+
+    def description(self):
+        return 'Regressions without regressed_by and some dependencies'
+
+    def handle_bug(self, bug, data):
+        bugid = bug['id']
+        deps = set(bug['blocks']) | set(bug['depends_on'])
+        assignee = bug['assigned_to_detail']
+        if utils.is_no_assignee(assignee['email']):
+            assignee = None
+
+        data[str(bugid)] = {
+            'deps': deps,
+            'assignee': assignee,
+            'creator': bug['creator_detail'],
+            'creation': dateutil.parser.parse(bug['creation_time']),
+        }
+        return bug
+
+    def filter_bugs(self, bugs):
+        all_deps = set()
+        dep_bug_creation = {}
+
+        for bugid, info in bugs.items():
+            bugid = int(bugid)
+            info['deps'] = deps = set(x for x in info['deps'] if x < bugid)
```

Comment:
Add a comment here explaining that we can ignore all bugs with IDs higher than the current one, as the current one couldn't possibly be caused by a bug that was filed after itself.